### PR TITLE
Feature/cloudinary

### PR DESCRIPTION
**Pull request by johnmeade-webdev**
The connection between cloudinary, the clientside, and the api is set. I have temporarily disabled the image filepath as 'required' in both the model and the joi file, just so we don't add cloudinary stuff without knowing that what I grab from them for the path is alright for populating our BeverageFeed. That's next on my list.

### CODE REVIEW

**Comment by johnmeade-webdev** [comment]
DON'T MERGE YET!

**Comment by johnmeade-webdev** [comment]
Ready :)

**Comment by SDBowen** [comment]
Nice!

File: api/validation/drink.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const drinkSchema = Joi.object().keys({
     .required(),
   tastingNotes: Joi.string().required(),
   comments: Joi.string().allow(""),
-  image: Joi.string().required(),
+  // image: Joi.string().required(),
+  image: Joi.string().allow(""),
```
Comment (SDBowen) [inline api/validation/drink.js:14]: Do we want to allow users to submit empty strings for the image?

**Review by SDBowen — APPROVED** [review]

File: api/validation/drink.js (hunk 1 above)
Comment (johnmeade-webdev) [reply api/validation/drink.js:14]: ugh. I had that part disabled for testing. I set it as required in the model, but not back to required in Joi.

**Pull request merged**